docker-images/development.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:bionic
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get -q update && apt-get -y dist-upgrade
+RUN apt-get install -yq --no-install-recommends \
+        cmake curl cmake gcc g++ git libmagic-dev \
+        libssl-dev pkg-config ca-certificates
+
+# Install rust
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly-2019-02-05
```

Comment:
The reason I pinned it is because I want a nightly release with a rustfmt and clippy build. Most nigthly builds doesn't contain both of them, either rustfmt or clippy is missing. We can simply use something like this:

```
RUN rustup component add rustfmt || cargo install --git https://github.com/rust-lang/rust-clippy clippy
```

This is fine for development environment since it will build only once when we run `docker-compose build` but it is really a time consuming operation for our CI since it always builds and uses same image.